src/adapters/sovrn.js

```diff
@@ -119,7 +119,7 @@ var SovrnAdapter = function SovrnAdapter() {
 
               //store bid response
               //bid status is good (indicating 1)
-              bid = bidfactory.createBid(1);
+              bid = bidfactory.createBid(1, bidObj);
```

Comment:
Can this be required/implemented for all adapters? This issue is not isolated to Sovrn, it is just potentially more commonly setup there and was noticed first. aardvark and triplelift are two additional examples where we will run into this issue in our current setup as we setup flex sizes for them.